api/conf/settings/security.py

```diff
@@ -32,7 +32,7 @@
         "0.0.0.0",
     ]
 
-BASE_URL = config("BASE_URL", default="https://openverse.org/")
+BASE_URL = config("BASE_URL", default="https://api.openverse.engineering/")
```

Comment:
I wonder why this variable is here instead of inside the `base.py` file at the same level 🤔 A curious thing!